app/routes/circulars/index.tsx

```diff
@@ -263,15 +269,41 @@ export default function () {
               </li>
             ))}
           </ol>
-          {totalPages > 1 && (
-            <Pagination
-              query={query}
-              page={page}
-              totalPages={totalPages}
-              startDate={startDate}
-              endDate={endDate}
-            />
-          )}
+          <div className="display-flex flex-row flex-wrap">
+            <div className="flex-align-start tablet:flex-fill">
+              <div>
+                <Label className="margin-top-auto" htmlFor="value">
+                  Results Per Page
+                </Label>
+                <Dropdown
+                  id="limit"
+                  className="usa-select height-4 padding-y-0"
+                  name="limit"
+                  defaultValue={limit}
+                  form="searchForm"
+                  onChange={({ target: { form, value } }) => {
+                    submit(form)
+                  }}
+                >
+                  <option value="100">100</option>
+                  <option value="250">250</option>
+                  <option value="500">500</option>
```

Comment:
Let's make 100 the _maximum_ number of results per page.

```suggestion
                  <option value="10">10</option>
                  <option value="20">20</option>
                  <option value="50">50</option>
                  <option value="100">100</option>
```

Also, we need to validate _on the server side_ that the requested number of results is less than an or equal to the maximum.

Reply:
> Let's make 100 the _maximum_ number of results per page.
> 
> Also, we need to validate _on the server side_ that the requested number of results is less than an or equal to the maximum.

would the default number of results now be 10 rather than 100?

as for the server side validation, are we essentially preventing more than 100 results being displayed?

Reply:
> would the default number of results now be 10 rather than 100?

I don't think so. 100 is still a good default.

> as for the server side validation, are we essentially preventing more than 100 results being displayed?

Yes.